src/main/java/jep/ClassList.java

```diff
@@ -159,6 +165,31 @@ private void loadClassPath() {
         }
     }
 
+   /**
+     * Recursively go through a folder and all subdirectories looking for .class
+     * files. Add them all.
+     */
+    private void addClassFilesInTree(File folder, String prefix) {
```

Comment:
Can you add javadoc `@param` to the javadoc on this method?  I realize ClassList.java is not very consistent with javadoc, but I'd like to improve the readabiity.  It took me a bit to figure out that prefix is going to be building a string of the package name as you walk down the directory tree, so maybe add a bit of explanation on that too?